src/app/dock_widgets/process/gt_processdock.cpp

```diff
@@ -1399,6 +1401,93 @@ GtProcessDock::renameElement()
     }
 }
 
+void
+GtProcessDock::moveElements(const QList<QModelIndex>& source,
+                            const QModelIndex& target)
+{
+    if (!target.isValid()) return;
+
+    if (source.isEmpty()) return;
```

Comment:
I would prefer merging such simple conditions
```suggestion
    if (!target.isValid() || source.isEmpty()) return;
```